homeassistant/components/light/zwave.py

```diff
@@ -244,9 +235,9 @@ def _value_added(self, value):
         # Check for the missing color values
         self._get_color_values()
 
-    def update_properties(self):
+    def update(self):
```

Comment:
Why would we ever overwrite update from a component ABC? We don't do any direct interaction with devices in the ABCs.